src/components/UI/Display/RefreshableLabel.js

```diff
@@ -51,7 +51,7 @@ function RefreshableLabel({ children, value, className }) {
   useEffect(() => {
     const sToMs = 1000;
     const compareData = () => {
-      if (prevValue && value !== prevValue) {
+      if (typeof prevValue !== 'undefined' && value !== prevValue) {
         setHasDataChanged(true);
       }
```

Comment:
`0` is being coerced to `false`, and subject counts changing from `0` to `1` was not appropriately being highlighted. I think by checking for `undefined` is more appropriate to check whether there is a previous value.